crates/router/src/services/authorization/permissions.rs

```diff
@@ -67,8 +67,32 @@ generate_permissions! {
             scopes: [Read, Write],
             entities: [Merchant]
         },
-        Recon: {
-            scopes: [Write],
+        ReconToken: {
+            scopes: [Read],
+            entities: [Merchant]
+        },
+        ReconFiles: {
+            scopes: [Read, Write],
+            entities: [Merchant]
+        },
+        ReconAndSettlementAnalytics: {
+            scopes: [Read],
+            entities: [Merchant]
+        },
+        ReconUpload: {
+            scopes: [Read, Write],
+            entities: [Merchant]
+        },
+        ReconReports: {
+            scopes: [Read, Write],
```

Comment:
What is the purpose of `Write` here? What will be operations on `Reports` that requires `Write`?